component-server-parent/component-server/src/test/java/org/talend/sdk/component/server/service/jcache/FrontCacheResolverTest.java

```diff
@@ -53,14 +57,18 @@ void cleanupCaches() {
     }
 
     @Test
-    void clearCaches() {
+    void clearCaches() throws JsonProcessingException {
         final int connectors = service.getConnectors().getPluginsList().size();
-        final Response resp = base
+        final javax.ws.rs.core.Response resp = base
```

Comment:
why do you decide to use fully qualified name instead of short one?

Reply:
You're right, I missed to remove it.